jupyter_book/create.py

```diff
@@ -242,17 +219,15 @@ def upgrade_book():
     try:
```

Comment:
I'm not a big fan of `try` statements ...

errors pass silently if you have them

Reply:
though in this case we're capturing errors and displaying a custom error message at the end if we run into the errors, no?

Reply:
This seems like an reasonable `try/except` block to me, but would be really interested in how you think it could be improved @jasmainak  !

Reply:
It was my priors commenting rather than the posterior :-P But wouldn't you hide problems in the upgrade if you use such a block? Imagine a user has a problem upgrading. When you ask them for the traceback, they won't have one because it's been caught and instead you will get the print statement -- which is not so useful for debugging.